src/elmExecutor/executeElm.js

```diff
@@ -8,10 +8,11 @@ function executeElm(smart, fhirVersion, request, executionInputs, consoleLog) {
   return new Promise(function(resolve, reject){
     console.log("about to executeElm()");
     const patientSource = getPatientSource(fhirVersion);
-    const neededResources = extractFhirResourcesThatNeedFetching(executionInputs.elm);
+    const neededResourcesFromLibrary = retrieveNeededResource(executionInputs.mainLibraryMaps[executionInputs.elm.library.identifier.id]);
+    //compareElmAndLibraryOutput(executionInputs, neededResourcesFromLibrary);
```

Comment:
Does it make sense to add an "s" to retrieveNeededResource?

retrieveNeededResource**s**

Reply:
Sure, just pushed the changes.